packages/suite/src/actions/wallet/__tests__/coinjoinAccountActions.test.ts

```diff
@@ -70,7 +70,12 @@ describe('coinjoinAccountActions', () => {
             testMocks.setTrezorConnectFixtures(f.connect);
             jest.spyOn(console, 'log').mockImplementation(() => {});
 
-            await store.dispatch(coinjoinAccountActions.createCoinjoinAccount(f.params as any)); // params are incomplete
+            await store.dispatch(
+                coinjoinAccountActions.createCoinjoinAccount(
+                    f.params.network as any,
+                    f.params.account as any,
+                ),
```

Comment:
Note: `@total-typescript/shoehorn` is a magic :magic_wand: fix that can help us get rid of `any`.

But not so easily in this case, because in the fixture, there are strings typed as `string` that would have to be cast as a type union, and it would be quite noisy.